Chapter3_Functions/Chapter 3 #2.js

```diff
@@ -0,0 +1,13 @@
+function isEven(n)
+{
+    if(n==0)
+    return true;
+    else if(n==1)
+    return false;
+    else if(n<0)
+    console.log("WRONG INPUT");
+    else
+    return isEven(n-2);
+}
+isEven(50);
+isEven(75);
```

Comment:
These need to be printed out to console.
console.log(isEven(50));
console.log(isEven(75));